modules/mpv-player/android/src/main/java/expo/modules/mpvplayer/nativeplayer/TrickplayProvider.kt

```
@@ -0,0 +1,139 @@
+                        readTimeout = 20000
+                        config.headers?.forEach { (k, v) -> setRequestProperty(k, v) }
+                    }
+                    conn.inputStream.use { input ->
+                        BitmapFactory.decodeStream(input)
+                    }
+                }
+                if (bitmap != null) {
+                    cache.put(index, bitmap)
+                }
+                bitmap
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)